internal/k8s/ako_init.go

```diff
@@ -760,8 +761,17 @@ func (c *AviController) FullSyncK8s(sync bool) error {
 					labelSelectorMap[nodeLabels["key"]] = nodeLabels["value"]
 				}
 			}
-			nodeObjects, _ := utils.GetInformers().NodeInformer.Lister().List(labels.Set(labelSelectorMap).AsSelector())
-			for _, node := range nodeObjects {
+			//send sorted list of nodes from here.
+			allNodes := objects.SharedNodeLister().CopyAllObjects()
```

Comment:
why are we dropping the node label selector here

Reply:
@apalsule: `allNodes` contains all node objects filtered with node-label.